tcrdist/repertoire.py

```diff
@@ -1371,9 +1414,9 @@ def _compute_pairwise(sequences, metric = "nw", processes = 2, user_function = N
     pw = pairwise._pack_matrix(pw)
     pw_df = pd.DataFrame(pw, index = unique_seqs, columns = unique_seqs)
     pw_full = pw_df.loc[sequences, sequences]
-    pw_full_np = pw_full.values
-    return(pw_full_np)
-
+    pw_full_np = pw_full.values"""
+    return pw_full_np
+    
```

Comment:
The final returned object is the `pw_full_np` which is coming from the new method:
```
dvec = pwseqdist.apply_pairwise_sq(sequences.values, metric_func,
                                    ncpus=processes, **kwargs)

 pw_full_np = scipy.spatial.distance.squareform(dvec)
```